rails.md

```diff
@@ -38,51 +38,53 @@ Answer the following questions. First, without external resources. Challenge you
 
 ```
 /users/       method="GET"     # :controller => 'users', :action => 'index'
-/users/1      method="GET"     # :controller => 'users', :action => 'show'
-/users/new    method="GET"     # :controller => 'users', :action => 'new'
+/users/1      method="GET"     # :controller => 'users', :action => 'show'== 1
+/users/new    method="GET"     # :controller => 'users', :action => 'new' 
 /users/       method="POST"    # :controller => 'users', :action => 'create'
-/users/1/edit method="GET"     # :controller => 'users', :action => 'edit'
-/users/1      method="PUT"     # :controller => 'users', :action => 'update'
-/users/1      method="DELETE"  # :controller => 'users', :action => 'destroy'
+/users/1/edit method="GET"     # :controller => 'users', :action => 'edit'== 2
+/users/1      method="PUT"     # :controller => 'users', :action => 'update'== 3
+/users/1      method="DELETE"  # :controller => 'users', :action => 'destroy'== 4
 ```
-
+Because they have a changing factor in the object you are controlling.
 
 
 
 
 4. What is the public folder used for in Rails?
 
-  Your answer:
+  Your answer: it holds javascript ,html and css files. basic files. 
 
-  Researched answer:
+  Researched answer: In many MVC frameworks (e.g. Rails), the public folder is used to serve static files. ... The role of the public folder is to have those views that would be publicly accessible in the application.
 
 
 
 5. Write a rails route for a controller called "main" and a page called "game" that takes in a parameter called "guess"
 
-
+get '/main/game/guess' => "main#main"
 
```

Comment:
get '/game/:guess' => 'main#game'
The page name 'game' would be the initial route, the parameter follows the route, the controller is main followed by the method